bullet/src/KinematicsFeatures.cc

```diff
@@ -0,0 +1,98 @@
+#include <ignition/common/Console.hh>
+#include "KinematicsFeatures.hh"
+
+namespace ignition {
+namespace physics {
+namespace bullet {
+
+/////////////////////////////////////////////////
+FrameData3d KinematicsFeatures::FrameDataRelativeToWorld(
+    const FrameID &_id) const
+{
+  FrameData3d data;
+
+  // The feature system should never send us the world ID.
+  if (_id.IsWorld())
+  {
+    ignerr << "Given a FrameID belonging to the world. This should not be "
+           << "possible! Please report this bug!\n";
+    assert(false);
+    return data;
+  }
+
+  const auto linkID = _id.ID();
+  const auto &linkInfo = this->links.at(linkID);
+  const auto modelIdentity = linkInfo->model;
+  // auto linkIndex = linkInfo->linkIndex;
+  const auto &modelInfo = this->models.at(modelIdentity);
+  const auto &model = modelInfo->model;
+
+  const btVector3 pos = model->getCenterOfMassTransform().getOrigin();
+  const btMatrix3x3 mat = model->getCenterOfMassTransform().getBasis();
+
+  auto eigenMat = convert(mat);
+  auto eigenVec = convert(pos);
+
+  data.pose.linear() = eigenMat;
+  data.pose.translation() = eigenVec;
+
+  // Get frame velocities
```

Comment:
Those comments can be removed, as those are already in the previous implementation in the case we ever need them.

Reply:
Removed on 25225aa